extensions/gateway/pkg/command/command.go

```diff
@@ -223,6 +224,7 @@ func spacesProviders(cfg *config.Config, logger log.Logger) map[string]map[strin
 	// generate rules based on default config
 	return map[string]map[string]interface{}{
 		cfg.StorageUsersEndpoint: {
+			"providerid": "1284d238-aa92-42ce-bdc4-0b0000009157",
```

Comment:
for now this is hardcoded ... we should generate the storageproviderid for the userstorageprovider on first start... furthermore, the userstorageprovider currently also stores project spaces ... so maybe just call it `storageprovider`?

Reply:
maybe with `ocis init` from https://github.com/owncloud/ocis/pull/3551

Reply:
Should we move it to `utils` package meanwhile? Then it is aligned with the `StorageProviderID` and `PublicStorageProviderID`?

Reply:
The userstorageproviderid is not used in the reva codebase, only in config files, so there is no point in adding a variable to revas `pkg/utils` or `resourceid` package that is unused.